heron/executor/src/python/BUILD

```diff
@@ -2,8 +2,15 @@ package(default_visibility = ["//visibility:public"])
 
 load("/tools/rules/pex_rules", "pex_library", "pex_binary")
 
+pex_library(
+    name = "heron-executor-lib",
+    srcs = ["heron_executor.py"],
+    deps = ["//heron/statemgrs/src/python:zk-statemgr-py"],
+    reqs = ["pyyaml==3.10"],
+)
+
 pex_binary(
     name = "heron-executor",
-    srcs = ["heron-executor.py"],
-    reqs = ["pyyaml==3.10"],
+    srcs = ["heron_executor.py"],
+    deps = [":heron-executor-lib"],
```

Comment:
That's basically what I had started with when I refactored to `heron_executor_core.py` with just the main in `heron-executor.py`. At the time I felt it complicated working with the code so I was glad to consolidate back into just `heron_executor.py` so I think one source file is preferred over removing the dup srcs in the build targets.